### PR TITLE
Define ENV variable to suspend regressions reporting for some testcases

### PR DESCRIPTION
**Pull request by s0b0lev**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -111,7 +111,8 @@ jobs:
                 --output-dir /opt/edelweiss \
                 --plugin-dir /opt/mythril \
                 --s3 \
-                --circle-ci CircleCI/mythril.csv
+                --circle-ci CircleCI/mythril.csv\
+                --ignore-regressions $IGNORE_REGRESSIONS
```
Comment (b-mueller) [inline .circleci/config.yml]: Change to $IGNORE_FALSE_POSITIVES?

File: .circleci/config.yml [hunk 2]
```diff
@@ -111,7 +111,8 @@ jobs:
                 --output-dir /opt/edelweiss \
                 --plugin-dir /opt/mythril \
                 --s3 \
-                --circle-ci CircleCI/mythril.csv
+                --circle-ci CircleCI/mythril.csv\
```
Comment (b-mueller) [inline .circleci/config.yml]: This file is hosted on S3 right?

Comment (s0b0lev) [reply .circleci/config.yml]: Yep, right

File: .circleci/config.yml (hunk 1 above)
Comment (s0b0lev) [reply .circleci/config.yml]: Changed

**Pull request merged**